main.go

```diff
@@ -14,13 +19,43 @@ var (
 
 func main() {
 	app := cli.NewApp()
-	app.Name = "rancher-vxlan"
 	app.Version = VERSION
 	app.Flags = []cli.Flag{
+		cli.StringFlag{
+			Name: "pid-file",
+		},
+		cli.StringFlag{
+			Name:  "listen",
+			Value: ":8111",
+		},
 		cli.BoolFlag{
 			Name:   "debug, d",
 			EnvVar: "RANCHER_DEBUG",
 		},
+		cli.BoolFlag{
+			Name:   "disable-tx-offload",
+			EnvVar: "RANCHER_DISABLE_TX_OFFLOAD",
+		},
+		cli.StringFlag{
+			Name:   "overlay-iface",
+			Value:  vxlan.OverlayInterfaceName,
+			EnvVar: "RANCHER_OVERLAY_IFACE",
+		},
+		cli.StringFlag{
+			Name:   "docker-bridge",
```

Comment:
I think we do need two flags. One for the docker bridge, the other one for the bridge where the vxlan containers are connected.